CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## Changelog
```

Comment:
I don't think it makes sense to have a changelog for the top-level package - that's just a wrapper for the other ones to make it easier to run the server. if we're going to start a changelog i would make it for the opds-web-client package.

Reply:
which I see you also did :)